source/Posets/Delta-V-CompletePoset.lagda

```diff
@@ -0,0 +1,759 @@
+            (λ P → λ Q → (P ⊑ Q , ⊑-is-prop-valued P Q)) ,
+            (((λ P → ⊑-is-reflexive P) , λ P → λ Q → λ R → ⊑-is-transitive P Q R) , λ o → λ r → ⊑-is-antisymmetric o r))
+
+ open Local-Smallness (𝓥 ⁺) 𝓥 𝓥 Ω-Poset (λ P → λ Q → (P ⊑ Q , ⊑-is-prop-valued P Q))
+
+ ⊑-is-locally-small : is-locally-small-order
+ ⊑-is-locally-small P Q = (P ⊑ Q , ≃-refl (P ⊑ Q))
+
+ open δ-complete-poset 𝓥 Ω-Poset
+
+ Ω-δ-complete : is-δ-complete
+ Ω-δ-complete Q R Q-⊑-R P = ((Ǝ i ꞉ (𝟙 + P holds) , (δ Q R P i holds)) , (is-upbnd , has-sup-cond))
+  where
+   open Joins (λ Q → λ R → (Q ⊑ R , ⊑-is-prop-valued Q R))
+   open propositional-truncations-exist pt
+
+   is-upbnd : ((Ǝ i ꞉ (𝟙 + P holds) , (δ Q R P i holds)) is-an-upper-bound-of ((𝟙 + P holds) , (δ Q R P))) holds
+   is-upbnd i e = ∣ (i , e) ∣
+
+   has-sup-cond : ((U , _) : upper-bound ((𝟙 + P holds) , (δ Q R P))) → (Ǝ i ꞉ (𝟙 + P holds) , (δ Q R P i holds)) ⊑ U
+   has-sup-cond (U , U-is-upbnd) = ∥∥-rec (holds-is-prop U) f
+    where
+     f : Σ i ꞉ (𝟙 + (P holds)) , δ Q R P i holds → U holds
+     f (i , e) = U-is-upbnd i e
+
+ open Positive-Posets (𝓥 ⁺) 𝓥 𝓥 Ω-Poset
+ open positive-posets Ω-δ-complete
+
+ Ω-positive : is-positive-poset
+ Ω-positive = (⊥ , ⊤ , (𝟘-elim , f))
+  where
+   open Joins (λ Q → λ R → (Q ⊑ R , ⊑-is-prop-valued Q R))
+   f : (Q : Ω 𝓥) → ⊤ ⊑ Q → (P : Ω 𝓥) → (Q is-lub-of ((𝟙 + (P holds)) , δ ⊥ Q P)) holds → P holds
+   f Q o P (Q-is-upbnd , Q-has-lub-cond) = Q-has-lub-cond (P , P-is-upbnd) (o ⋆)
+    where
+     P-is-upbnd : (P is-an-upper-bound-of ((𝟙 + (P holds)) , δ ⊥ Q P)) holds
+     P-is-upbnd (inr p) e = p
+
+module Ω¬¬-δ-complete-non-trivial-Poset (𝓥 : Universe) where
+
+ _⊑_ : Ω¬¬ 𝓥 → Ω¬¬ 𝓥 → 𝓥  ̇
+ P ⊑ Q = P holds' → Q holds'
+
+ ⊑-is-prop-valued : (P Q : Ω¬¬ 𝓥) → is-prop (P ⊑ Q) 
+ ⊑-is-prop-valued P Q = Π-is-prop fe (λ _ → holds'-is-prop Q)
+
+ ⊑-is-reflexive : (P : Ω¬¬ 𝓥) → P ⊑ P
+ ⊑-is-reflexive _ = id
+
+ ⊑-is-antisymmetric : {P Q : Ω¬¬ 𝓥} → P ⊑ Q → Q ⊑ P → P ＝ Q 
+ ⊑-is-antisymmetric {P} {Q} o r =
+   to-subtype-＝ (λ X → being-¬¬-stable-is-prop fe (holds-is-prop X))
+                 (to-subtype-＝ (λ _ → being-prop-is-prop fe) (pe (holds'-is-prop P) (holds'-is-prop Q) o r))
+
+ ⊑-is-transitive : (P Q R : Ω¬¬ 𝓥) → P ⊑ Q → Q ⊑ R → P ⊑ R
+ ⊑-is-transitive P Q R o r p = r (o p)
+
+ Ω¬¬-Poset : Poset (𝓥 ⁺) 𝓥
+ Ω¬¬-Poset = (Ω¬¬ 𝓥 ,
+            (λ P → λ Q → (P ⊑ Q , ⊑-is-prop-valued P Q)) ,
+            (((λ P → ⊑-is-reflexive P)
+              , λ P → λ Q → λ R → ⊑-is-transitive P Q R)
+              , λ o → λ r → ⊑-is-antisymmetric o r))
+
+ open Local-Smallness (𝓥 ⁺) 𝓥 𝓥 Ω¬¬-Poset (λ P → λ Q → (P ⊑ Q , ⊑-is-prop-valued P Q))
+
+ ⊑-is-locally-small : is-locally-small-order
+ ⊑-is-locally-small P Q = (P ⊑ Q , ≃-refl (P ⊑ Q))
+
+ open δ-complete-poset 𝓥 Ω¬¬-Poset
+
+ Ω¬¬-δ-complete : is-δ-complete
+ Ω¬¬-δ-complete Q R Q-⊑-R P =
+   (((¬¬ (((Ǝ i ꞉ (𝟙 + P holds) , (δ Q R P i) holds') holds)) , negations-are-props fe) , ¬-is-¬¬-stable) , (is-upbnd , has-lub-cond)) 
```

Comment:
This line got a little too long.

Reply:
Hmm it didn't look too bad to me, but where should I line break?

Reply:
Just before `(is-upbnd , has-lub-cond)` is very natural, since it's a new component anyway.
You could even break several times, e.g. introduce the type on one line, show it's a prop on the next, show it's not-not-stable on the line after that, etc.

Reply:
Try to stick to max. 80 characters in width.